azure-pipelines.yml

```diff
@@ -10,6 +10,8 @@ jobs:
         # TODO: RuboCop needs fixed in
         # https://github.com/Homebrew/homebrew-bundle/pull/384
         # brew style homebrew/bundle
+        # Use local config until then
+        rubocop --force-exclusion --parallel --except FormulaAudit,FormulaAuditStrict,NewFormulaAudit --format simple
```

Comment:
@colindean This seems like it's maybe not working as expected: https://dev.azure.com/Homebrew/Homebrew/_build/results?buildId=2783

It's outputting failures but not failing the build.

Reply:
Running `rubocop --force-exclusion --parallel --except FormulaAudit,FormulaAuditStrict,NewFormulaAudit --format simple` locally exits 1. 

I wonder if this bash block needs a `set -e` at the beginning? I'll give it a shot.

Reply:
When I run `brew style homebrew/bundle`, those alignment warnings aren't present. Maybe `.rubocop.yml` in this repo and the main repo have diverged further. If I run autofix with the local `.rubocop.yml`, the code style will then be in conflict with upstream. I don't think we want that. If we add `set -e`, then the current state will fail and we won't be able to merge anything until we've accepted that conflict or completed #388…

Reply:
https://github.com/Homebrew/homebrew-bundle/pull/436 makes it exit but I'm not sure we should merge that yet.

Reply:
See #437.

Reply:
Thanks ❤️